website_snippet_product_category/views/snippets.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="snippets" inherit_id="website.snippets">
+        <xpath
+            expr="//div[@id='snippet_feature']//div[hasclass('o_panel_body')]"
+            position="inside"
+        >
+            <t
+                t-snippet="website_snippet_product_category.s_product_category"
+                t-thumbnail="/website_snippet_product_category/static/src/img/s_product_category.png"
+            />
+        </xpath>
+    </template>
+    <!-- Snippet Product Category -->
+    <template id="s_product_category" name="Product Category">
+        <section class="s_product_category py-4">
+            <div class="js_product_category col-12" />
+        </section>
+    </template>
+    <template id="s_product_category_items_recursive">
+        <t
+            t-foreach="categories"
+            t-as="category"
+            t-if="categories and cur_level &lt;= 4"
+        >
+            <div
+                t-attf-class="row categ_tree_level {{cur_level == 1 and 'mb-2 main_tree_level'}} {{cur_level &gt; 1 and 'pl-'+str(min((cur_level-1)*2, 5))}}"
+                t-att-data-tree-level="cur_level"
+                t-if="category.website_published"
+            >
+                <div class="col-12">
+                    <div
+                        t-if="category.website_image"
+                        t-field="category.website_image"
+                        t-attf-class="categ_img d-inline-block pr-1"
+                        t-options="{'widget': 'image', 'class': 'img img-responsive rounded','resize': '24x24' }"
+                    />
+                    <a
+                        t-att-href="keep('/shop/category/' + slug(category), category=0)"
+                        class="categ_link"
+                        t-field="category.name"
```

Comment:
This does not work as a normal t-field and you cannot edit category names from the widget.

I'm not sure if that's expected or just an accident. It's not like a big deal.

Reply:
The content of the snippet is not editable with the web editor by design (only can use the snippet options to change the style)